### PR TITLE
fix: slack integration card layout

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: #5730

### Description:

before:
image

after:
CleanShot 2023-06-05 at 18 29 59@2x
CleanShot 2023-06-05 at 18 31 35@2x
CleanShot 2023-06-05 at 18 31 52@2x

### CODE REVIEW

File: packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx [hunk 1]
```diff
@@ -29,68 +32,87 @@ const SlackSettingsPanel: FC = () => {
...
+                        </Group>
+
+                        {data && !hasRequiredScopes(data) && (
+                            <Alert
+                                color="blue"
+                                icon={<MantineIcon icon={IconAlertCircle} />}
+                            >
+                                Your Slack integration is not up to date, you
+                                should reinstall the Slack integration to
+                                guaranty the best user experience.
```
Comment (joaoviana) [inline packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx:98]:
```suggestion
                                Your Slack integration is not up to date, you
                                should reinstall the Slack integration to
                                guarantee the best user experience.
```

File: packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx [hunk 2]
```diff
@@ -29,68 +32,87 @@ const SlackSettingsPanel: FC = () => {
...
     }
 
     const isValidSlack = data?.slackTeamName !== undefined && !isError;
     return (
-        <SlackSettingsWrapper>
-            <SlackIcon src={slackSvg} />
-            <AppearancePanelWrapper>
-                <Title> Slack integration</Title>
+        <SettingsGridCard>
+            <div>
```
Comment (joaoviana) [inline packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx:41]: I was thinking about this the other day, do we want to replace `div`s with `Box` component or we don't care much about that in this case? 🤔

File: packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx [hunk 3]
```diff
@@ -29,68 +32,87 @@ const SlackSettingsPanel: FC = () => {
...
-                    </Description>
-                )}
+                    <Stack spacing="xs">
+                        {isValidSlack && (
+                            <Text color="dimmed">
+                                Added to the{' '}
+                                <Text span fw={500} color="black">
+                                    {data.slackTeamName}
+                                </Text>{' '}
+                                slack workspace.
```
Comment (joaoviana) [inline packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx:55]:
```suggestion
                                Slack workspace.
```

**Review by joaoviana — CHANGES_REQUESTED** [review]
🎉 
one minor typo

File: packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx (hunk 2 above)
Comment (IrakliJani) [reply packages/frontend/src/components/UserSettings/SlackSettingsPanel/index.tsx:41]: if there's nothing mantine specific then its just a unnecessary complexity

**Review by joaoviana — APPROVED** [review]

**Pull request merged**